services/datcore-adapter/src/simcore_service_datcore_adapter/modules/pennsieve.py

```diff
@@ -52,9 +35,50 @@ def _compute_file_path(
     return Path(file_path)
 
 
+class PennsieveAuthorizationHeaders(TypedDict):
+    Authorization: str
+
+
+_TTL_CACHE_AUTHORIZATION_HEADERS_SECONDS: Final[
+    int
+] = 3530  # pennsieve authorizes 3600 seconds
```

Comment:
MINOR: add here reference to pennsieve relevant doc? you also mention recipes .. add refs here please